### PR TITLE
Sweep all

### PR DESCRIPTION
**Pull request by TheCharlatan**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -667,6 +667,23 @@ impl WalletClient {
         Ok(monero::PrivateKey::from_slice(&rsp.key.0)?)
     }
 
+    /// Return the view private key.
+    pub async fn query_spend_key(&self) -> anyhow::Result<monero::PrivateKey> {
```
Comment (h4sh3d) [inline src/lib.rs:671]: Doc comment is not correct

File: src/models.rs [hunk 2]
```diff
@@ -234,6 +234,33 @@ pub struct IncomingTransfer {
     pub tx_hash: HashString<CryptoNoteHash>,
     pub tx_size: u64,
 }
+#[derive(Clone, Debug, Serialize, Deserialize)]
```
Comment (h4sh3d) [inline src/models.rs:237]: Missing a line between structs :p

**Review by h4sh3d — COMMENTED** [review]
What do you think about making some kind of `PrivateKeyType` for `Spend` and `View` and only one function that can query a key?

File: src/models.rs (hunk 2 above)
Comment (TheCharlatan) [reply src/models.rs:238]: :(

**Comment by TheCharlatan** [comment]
I merged the two key query functions into as per your suggestion @h4sh3d

**Pull request merged**